Tasks/NuGetToolInstallerV0/task.json

```diff
@@ -28,7 +28,7 @@
             "name": "versionSpec",
             "type": "string",
             "label": "Version of NuGet.exe to install",
-            "defaultValue": "4.3.0",
+            "defaultValue": "4.8.2",
```

Comment:
I should note that with this change the newer cred provider will be used https://github.com/Microsoft/artifacts-credprovider#nuget
I don't see a problem with it.